### DISCUSSION TITLE
How to user checkpoint in Flink-cdc-3.0.1

### DISCUSSION THREAD

**Discussion by lliuql** [apache/flink-cdc#3074] [Q&A]
Start a task as following:
./bin/flink-cdc.sh ./conf/mysql-2-doris.yaml

Question:
When start a task,How to specify checkpoint?

**Comment by Mrzyxing** [comment]
FYI： ./bin/flink-cdc.sh mysql_to_paimon.yaml --flink-home /opt/app/flink -Dexecution.checkpointing.interval=5s
